### PR TITLE
Update translations

### PR DESCRIPTION
**Pull request by plettich**
- Fix some issues with translations (spelling, marks)
- update translation templates for UI and server

### CODE REVIEW

**Comment by plettich** [comment]
@cornelinux the `*.po` and `*.pot` file are probably not important here but i fixed some spelling and how the  translation was tagged in the code.

**Comment by plettich** [comment]
@cornelinux the translation from the server for the token types works just fine.

File: privacyidea/static/components/token/views/token.import.html [hunk 1]
```diff
@@ -49,7 +49,7 @@ <h3 translate>Import tokens from a file</h3>
             The file is supposed to contain one token per line.
         </p>
         <p class="help-block" translate>
-            If you enrolling the Yubikey in HOTP mode, you should use the
+            If you are enrolling the Yubikey in HOTP mode, You should use the
```
Comment (cornelinux) [inline privacyidea/static/components/token/views/token.import.html]: "You should use" shouldn't it be a lower case "y"?

**Review by cornelinux — APPROVED** [review]

File: privacyidea/static/components/token/views/token.import.html (hunk 1 above)
Comment (cornelinux) [reply privacyidea/static/components/token/views/token.import.html]: @plettich I took the liberty to change this.

Comment (plettich) [reply privacyidea/static/components/token/views/token.import.html]: yeah, i guess i am old fashioned, i always capitalize the form of address :-)

**Pull request merged**